docs/examples/bicepsettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Linter": {
+    "Rules": {
+      "BCPL1010": {
+        "EnabledForEditing": true,

```

Comment:
>EnabledForEditing [](http://example.com/codeflow?start=9&length=17)

[MVP] I thought we removed these? I guess the docs section wasn't updated with the latest format for the config file?

Reply:
I'll fix this.

Reply:
fixed